README.md

```diff
@@ -1,21 +1,26 @@
 [![Build Status](https://semaphoreci.com/api/v1/mbta/dotcom/branches/master/badge.svg)](https://semaphoreci.com/mbta/dotcom)
+
 # Dotcom
 
 The new face of https://www.mbta.com/
-  - [Getting Started](#getting-started)
-  - [Running the Server](#running-the-server)
-  - [Environment Variables](#environment-variables)
-    - [`V3_API_KEY`](#v3apikey)
-    - [`V3_URL`](#v3url)
-    - [`GOOGLE_API_KEY`](#googleapikey)
-    - [`DRUPAL_ROOT`](#drupalroot)
-    - [`ALGOLIA_APP_ID`, `ALGOLIA_SEARCH_KEY`, and `ALGOLIA_ADMIN_KEY`](#algoliaappid-algoliasearchkey-and-algoliaadminkey)
+
+- [Getting Started](#getting-started)
+- [Running the Server](#running-the-server)
+- [Environment Variables](#environment-variables)
+    - [`V3_API_KEY`](#v3_api_key)
+    - [`V3_URL`](#v3_url)
+    - [`GOOGLE_API_KEY`](#google_api_key)
+    - [`DRUPAL_ROOT`](#drupal_root)
+    - [`ALGOLIA_APP_ID`, `ALGOLIA_SEARCH_KEY`, and `ALGOLIA_ADMIN_KEY`](#algolia_app_id-algolia_search_key-and-algolia_admin_key)
     - [Making the variables available to the app](#making-the-variables-available-to-the-app)
-  - [Additional documentation](#additional-documentation)
+- [Additional documentation](#additional-documentation)
+
 ## Getting Started
 
-1. Request a V3 API key at https://api-v3.mbta.com/. Note that, at any given time, the site may not be compatible with the
-very latest API version. As of this writing, the site is compatible with API version 2019-04-05.
+1. Request a V3 API key at https://dev.api.mbtace.com/, and request an increased
+rate limit for it (someone with access will need to approve this). Note that, at
+any given time, the site may not be compatible with the very latest API version.
+As of this writing, the site is compatible with API version `2019-07-01`.
```

Comment:
Too specific - I'd leave this last line out.

Reply:
The diff doesn't show it well, but this sentence is directly from the current README which also specifies a version, albeit an older one. I updated it with the reasoning that all of my tests passed when using the new version. If we want to delete the version entirely I'm in favor 🙂

Reply:
I'm in favor of removing info that gets dated quickly.